### ISSUE TITLE
The README that gets published on PyPi does not have images

### ISSUE THREAD

**Issue by dccote** [DCC-Lab/RayTracing#70]
If you look up the PyPi raytracing page, the figures don't show up. I think the links to the images in the main README.md should be absolute URL, and point directly to the master branch on GitHub.

**Comment by dccote** [comment]
I used absolute URLs, but also when linking to GitHub files, we must use `raw` not `blob`. I had to use "master" in the link, so we will only be able to confirm this is working fine once it is merged in master.

**Comment by PyMarc2** [comment]
>This is the only thing that worked for me. In markdown, a link would look like this: 
>![](https://raw.githubusercontent.com/username/project/branch/image.png)

This response is from : https://stackoverflow.com/questions/41983209/how-do-i-add-images-to-a-pypi-readme-that-works-on-github

Someone has to change this manually I suppose. I could do it.

**Comment by dccote** [comment]
Sovled with separate pull request #143
